src/config/index.ts

```diff
@@ -104,6 +105,13 @@ export const WAITING_FOR_PRODUCT_OWNER_LABEL = 'Waiting for: Product Owner';
 export const MAX_TRIAGE_DAYS = 2;
 export const MAX_ROUTE_DAYS = 1;
 
+/**
+ * Issues Someone Else Cares About Project
+ */
+
+export const ISSUES_PROJECT_NODE_ID = "PVT_kwDOABVQ184AOGW8";
+export const PRODUCT_AREA_FIELD_ID = "PVTSSF_lADOABVQ184AOGW8zgJEBno";
```

Comment:
How are you dev/testing? I know we hard-code other things like Slack channel IDs. Are those overridable somehow for local envs?

Reply:
I've just been replacing the value of these variables. It'd probably make my life easier if I include a way to override these defaults if env variables are defined for dev/testing purposes

Reply:
I can put up a PR for that after this goes in

Reply:
Can you add a quick commit here? I'm seeing plenty of prior art elsewhere in this file using `process.env`.

Reply:
done